app/controllers/claims/support/schools/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Claims::Support::Schools::UsersController < Claims::Support::ApplicationController
+  before_action :set_school
+
+  def index
+    @users = @school.users.claims
+  end
+
+  def show
+    @user = Claims::User.find(params.require(:id))
+  end
+
+  def new
+    @user = Claims::User.new
+  end
+
+  def check
+    if user.valid?
+      @user = user.decorate
+    else
+      render :new
+    end
+  end
+
+  def create
+    if UserInviteService.call(user, @school, sign_in_url)
+      redirect_to claims_support_school_users_path(@school)
+      flash[:success] = "User added"
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def set_school
```

Comment:
This set school method is replicated in every "School" namespaced controller.
Is it with separating this out into a concern and including it?
That way should you need to change it (for what ever reason), you don't need to change it X times.

Reply:
Hmm, I thought about it then decided against due to the clarity of containing everything within the individual controllers themselves.

But with the appropriate name, maybe it's clear enough. What do you think of below?

```ruby
class Claims::Support::Schools::UsersController
  include BelongsToSchool
end
```

Reply:
I think that would make a lot of sense.

Reply:
Added :)